src/components/ContentHighlights/HighlightStepper/HighlightStepperConfirmContent.jsx

```diff
@@ -104,10 +130,11 @@ const SelectedContent = () => {
       filterString += ')';
     }
     return filterString;
+  // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [currentSelectedRowIds]);
 
   if (currentSelectedRowIds.length === 0) {
-    return null;
+    return (<div data-testid="selected-content-no-results" />);
```

Comment:
Similar to another comment, rather than rendering an empty div here, it may be better to render _something_ in the UI (e.g., in an error warning `Alert`: "You don't have any highlighted content selected. Go back to the previous step to select content.").